controllers/storagecluster/storageclient.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 const (
-	ocsClientConfigMapName = "ocs-client-operator-config"
-	manageNoobaaSubKey     = "manageNoobaaSubscription"
+	ocsClientConfigMapName         = "ocs-client-operator-config"
+	manageNoobaaSubKey             = "manageNoobaaSubscription"
+	csiControllerPluginHostNetwork = "useHostNetworkForCsiControllers"
```

Comment:
Please have the var name match the value 
```suggestion
	useHostNetworkForCsiControllers = "useHostNetworkForCsiControllers"
```